File: c#/state-template/src/dominio/estados/EstadoConcretoA.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace state_template.src.dominio.estados
{
    class EstadoConcretoA : EstadoContexto 
    {
        public void actuar(Contexto contexto) 
        {
            //TODO
            if (this.cumpleCondicionParaCambiarDeEstado(contexto))
            {
                contexto.cambiarEstado(new EstadoConcretoB());
            }
        }

        private bool cumpleCondicionParaCambiarDeEstado(Contexto contexto)
        {
            //TODO
            return true;
        }
    }
}

File: python/src/estados/estado_concreto_a.py
from src.estados.estado_contexto import EstadoContexto
from src.estados.estado_concreto_b import EstadoConcretoB


class EstadoConcretoA(EstadoContexto):

    def actuar(self, contexto):
        # TODO
        if self.__cumple_condicion_para_cambiar_de_estado(contexto):
            contexto.cambiar_estado(EstadoConcretoB())

    def __cumple_condicion_para_cambiar_de_estado(self, contexto):
        # TODO
        return True